dash-pipeline/README-dash-as-submodule.md

```diff
@@ -0,0 +1,146 @@
+**C'Mon, I want a [Quick-Start](#quick-start)!**
+
+**Table of Contents**
+
+- [Importing the DASH project into another project](#importing-the-dash-project-into-another-project)
+- [Quick-Start](#quick-start)
+- [How to use DASH as a Git Submodule](#how-to-use-dash-as-a-git-submodule)
+- [Third-Party Workflow & DASH Workflow Reuse](#third-party-workflow--dash-workflow-reuse)
+  - [Recap: DASH bmv2 workflow](#recap-dash-bmv2-workflow)
+  - [Custom DASH Workflow](#custom-dash-workflow)
+  - [Reusable Build toolchain and artifacts](#reusable-build-toolchain-and-artifacts)
+  - [Required Custom Tools and Artifacts](#required-custom-tools-and-artifacts)
+  - [Custom Traffic Test Harness](#custom-traffic-test-harness)
+  - [Custom Tests](#custom-tests)
+  - [Third-Party CI Pipeline Automation (Git Actions)](#third-party-ci-pipeline-automation-git-actions)
+# Importing the DASH project into another project
+
+The [Azure/DASH project](https://github.com/Azure/DASH) can be used as a resouce within other projects, such as third-party, commercial or open-source DASH implementations. For example, a commercial DPU vendor can incorporate the DASH project into a private Git repository and utilize many of the components, providing consistency with the community implementation and definition, reusing test-cases, and avoiding duplication of efforts.
```

Comment:
resouce -> resource

Reply:
Thanks, fixed (along with many other typos and buzzwords once I merged from main and got Spellcheck running!).